### PR TITLE
Release Process Updates

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

**Comment by jasonblais** [comment]
+1

**Comment by esethna** [comment]
Thanks @jasonblais , tweaks made

**Comment by jasonblais** [comment]
+1

@lindy65 Can you help review so we can merge it by end of day before T-10 tomorrow?

File: source/process/release-process.md [hunk 1]
```diff
@@ -80,14 +79,13 @@ Exceptions can be made by the release manager setting priority to "Highest" and
 5. Build:  
     - Master is tagged and branched and “Release Candidate 1″ is cut (e.g. 1.1.0-RC1) according to the Release Candidate Checklist in ``mattermost/process``
     - CI servers are updated to the release branch 
-    - Size of bundle is checked
     - Directory structure is reviewed and large changes posted to the team channel
 6. PM:  
-    - Create meta issue for regressions in GitHub (see [example](https://github.com/mattermost/platform/issues/574))  
-    - PM owners for each new feature prepare a section highlighting end user benefits for the blog post and post to the Marketing room, including screenshots. See [example thread](https://pre-release.mattermost.com/core/pl/3mw3dfugutndipp1a8gzczqj6c) for formatting.
+    - PM owners for each new feature prepare a section highlighting end user benefits for the blog post and post to the Marketing room, including screenshots. See [example thread](https://pre-release.mattermost.com/core/pl/o611i4wz3pfafb6fpha9ggxxnh) for formatting.
```
Comment (lindy65) [inline source/process/release-process.md]: `end user` > `end-user`?

File: source/process/release-process.md [hunk 2]
```diff
@@ -80,14 +79,13 @@ Exceptions can be made by the release manager setting priority to "Highest" and
...
+        - PMs ensure tweet banners are speced and all assets are passed to the artist.
```
Comment (lindy65) [inline source/process/release-process.md]: `speced` > `spec'd`?

File: source/process/release-process.md [hunk 3]
```diff
@@ -123,9 +122,8 @@ Exceptions can be made by the release manager setting priority to "Highest" and
 7. Logistics:
     - Test RC fixes as they come in on CI servers
 8. Build: 
-    - Push next RC to acceptance after testing is complete and approved fixes merged, announces in Town Square on pre-release.mattermost.com/core
+    - Push next RC to acceptance after testing is complete and approved fixes merged, announces in Town Square. Build manager verifies with release manager before cutting any new RCs.
```
Comment (lindy65) [inline source/process/release-process.md]: Perhaps would read better worded as:
`Push next RC to acceptance after testing is complete and approved fixes are merged. Announce in Town Square. Build manager verifies with release manager before cutting any new RCs.`

File: source/process/release-process.md [hunk 4]
```diff
@@ -138,16 +136,21 @@ The final release is cut. If an urgent and important issue needs to be addressed
...
-    - Close final GitHub RC meta ticket  
+    - Close GitHub meta ticket for the release.
+    - Merge the docs release branch to master and verify all changes on docs.mattermost.com once the build is up. Submit a correction PR for any incorrect formatting or other errors missed during the intitial review.
+    - Update [MVP page](https://www.mattermost.org/mvp/) with the most valuable contributor of the release.
+    - Submit [GitLab MR](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/871/diffs) to take next Mattermost version in the Omnibus.
+    - Update [Docker preview image to latest version](https://github.com/mattermost/mattermost-docker-preview/commit/1c34195d70b26bd4c82b5ef4fa0ebaf421096881).
 4. Marketing:
     - Finalize Mailchimp email blast
-    - Finalize blog post for mattermost.org set timer for 06:00 PDT on first non-holiday weekday morning after release
+    - Finalize blog post for mattermost.org set timer for 06:00 PDT on the day of release.
```
Comment (lindy65) [inline source/process/release-process.md]: Perhaps add `and` e.g.
`Finalize blog post for mattermost.org and set timer for 06:00 PDT on the day of release.`

File: source/process/release-process.md [hunk 5]
```diff
@@ -187,6 +187,12 @@ If a bug fix release is required, run through the following steps again:
         - [Upgrade GitLab Mattermost to RC1](https://mattermost.atlassian.net/browse/PLT-3116)
         - [Push final build to GitLab Mattermost](https://mattermost.atlassian.net/browse/PLT-3117)
         - [Cut build and set up RC1 servers](https://mattermost.atlassian.net/browse/PLT-3115)
+    - Create [PR tracking spreadsheet](https://docs.google.com/spreadsheets/d/1YkRqyQk0Y4ZouV-SsXbSjbzGOXu2ZSPSd4XC_4LAErI/edit#gid=0) for next release.
+        - Populate with any PR's that have already went in for the next release.
```
Comment (lindy65) [inline source/process/release-process.md]: Perhaps reword:
`Populate with any PR's that have already been submitted for the next release.`

**Comment by esethna** [comment]
Tweaks made, thanks LI

**Pull request merged**